### DISCUSSION TITLE
Cannot log in because Github account email became inaccessible

### DISCUSSION THREAD

**Discussion by super-brian** [community/community#106965] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Not sure this is the best place to post, but I am in an usual situation.
To make everything short, the startup company that I worked for was sold to another company for which I work as a contractor.
(When the company was sold, I got a new job at a different company)
Now, the new owner of the company wants me to work for the product (mobile apps) of the startup company, but the private repos are in an account whose email is inaccessible. (email subscription is expired. email would NOT be deliverable.)
I have id/pw for the github account, and it requires email verification which is impossible because the email cannot be opened. What can I do? I sent a message to Github support, and it looks like it would take many days to get response, so here I ask.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


God bless you!
